run_test.py

```diff
@@ -35,6 +35,11 @@
 ]
 
 
+def _get_job_name():
+    # Returns Jenkins job name
+    return os.environ['JOB_NAME']
```

Comment:
How about
```suggestion
    return os.environ.get('JOB_NAME')
```
to support manual tests?

Reply:
Sorry.  `argconfig.py` uses `os.getenv` instead.
```suggestion
    return os.getenv('JOB_NAME')
```

Reply:
Fixed in 2fabab1d72cc0c048fcb0159152a6e0f77f2d755. Thanks.